docs/getting-started/integrations/dotnet.md

```diff
@@ -9,52 +9,27 @@ menu-title: .NET
 
 As a pure JavaScript/TypeScript application, CKEditor&nbsp;5 will work inside any environment that supports such components. While we do not offer official integrations for any non-JavaScript frameworks, you can include a custom build of CKEditor&nbsp;5 in a non-JS framework of your choice, for example, Microsoft's [.NET](https://dotnet.microsoft.com/).
 
-To integrate CKEditor&nbsp;5 with .NET, we will create a custom CKEditor&nbsp;5 build using the [online builder](https://ckeditor.com/ckeditor-5/online-builder/), and then import it into the .NET project.
+## Using the Builder
```

Comment:
Using CKEditor 5 Builder or Using Builder. This needs to change everywhere.

Reply:
We have 26 results for Builder, but only 8 out of these are CKEditor 5 Builder.

Reply:
I can unify the occurrences to "CKEditor 5 Builder." For me, it sounds better than bare-bones "Builder."

Reply:
It's ok to use Builder in the context of CKE5, but then don't use `the`.

Reply:
Can we, however, make it on epic, to no longer postpone this branch?